src/pybites_search/base.py

```diff
@@ -9,15 +10,19 @@
 from rich.text import Text
 
 ONE_DAY_IN_SECONDS = 24 * 60 * 60
-CACHE_EXPIRATION_SECONDS = config(
-    "CACHE_EXPIRATION_SECONDS", default=ONE_DAY_IN_SECONDS
-)
 TIMEOUT = 5
 
 console = Console()
 error_console = Console(stderr=True, style="bold red")
 
-requests_cache.install_cache("search_cache", expire_after=CACHE_EXPIRATION_SECONDS)
+HOME_DIR = str(Path.home())
```

Comment:
Interestingly was using TMP first, but that got flagged by flake8 (S108) then went with tempfile.gettempdir() but then the file might not persist. Turns out I had already written in changelog ("document driven design" haha) that we would default to user's home which makes more sense.